### DISCUSSION TITLE
New modules: Prometheus/VictoriaMetrics/Grafana

### DISCUSSION THREAD

**Discussion by tomershafir** [testcontainers/testcontainers-java#7132] [Ideas]
Hey, I would like to suggest new modules. 

As part of my work at Wix, I wrote a jvm app that scrapes VictoriaMetrics (Prometheus-compatible) through Grafana, and encountered the need in Prometheus/VictoriaMetrics/Grafana test containers for integration testing.

These days, such a stack is a popular subset within monitoring solutions, and all kind of companies write all kind of apps around it. I would love to hear your thoughts, rather then thoroughly examine the checklist in CONTRIBUTING upfront.

**Comment by eddumelendez** [comment]
Hi @tomershafir, 

Thanks for the offer! 

Please check the [docs for contributing](https://java.testcontainers.org/contributing/)

> Our ability to support the module in the future, potentially after contributors have moved on.

So, I would like to ask what is preventing you to publish the modules on your own? It can even be listed in the [Testcontainers Module Catalog](https://testcontainers.com/modules/)

**Reply by tomershafir** [reply]
@eddumelendez I don't have such atm. If you do think the use cases justify testcontainers modules, I can help contributing :) pls let me know

**Reply by tomershafir** [reply]
+grafana pyroscope module
@eddumelendez wdyt?
